globus_cli/commands/endpoint/server/list.py

```diff
@@ -14,6 +17,28 @@ def server_list(endpoint_id):
     """
     client = get_client()
 
+    endpoint = client.get_endpoint(endpoint_id)
+
+    if endpoint['host_endpoint_id']:  # not GCS -- this is a share endpoint
+        raise click.UsageError(
+            "{id} ({display_name}) is a share and does not have servers.\n"
+            "\n"
+            "To see details of the share, use\n"
+            "    globus endpoint show {id}\n"
+            "\n"
+            "To list the servers on the share's host endpoint, use\n"
+            "    globus endpoint server list {host_endpoint_id}".
+            format(**endpoint.data)
+        )
+
+    if endpoint['s3_url']:  # not GCS -- this is an S3 endpoint
+        if outformat_is_json():
+            safeprint(json.dumps({'s3_uri': endpoint['s3_url']}, indent=2))
```

Comment:
I noticed that this is `s3_uri` vs. `s3_url`.
Assuming this is just the result of being casual, can we use `s3_url` to be consistent with the API?